From 4d0fc3374413d8c893eba9bebe3d75b7d0c41a32 Mon Sep 17 00:00:00 2001
From: Pavel <texort@gmail.com>
Date: Sun, 3 Jul 2016 15:23:21 +0300
Subject: [PATCH] Check that _MSC_FULL_VER is defined

---
 date.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/date.h b/date.h
index 7922d57..ce67275 100644
--- a/date.h
+++ b/date.h
@@ -892,7 +892,7 @@ trunc(const std::chrono::duration<Rep, Period>& d)
 }
 
 // VS Update 2 provides floor, ceil, round, abs in chrono.
-#if _MSC_FULL_VER < 190023918
+#if !defined(_MSC_FULL_VER) || _MSC_FULL_VER < 190023918
 
 // round down
 template <class To, class Rep, class Period>
-- 
2.9.0

